### PR TITLE
Feature/form user input

### PR DESCRIPTION
**Pull request by ectweitmann**
### What (if any) features are you implementing?
-  The user receives a specific error message for the input fields they leave empty.
-  Users are only able to enter numbers within the minutes and seconds input fields.
### What (if anything) did you refactor?
- No
### Were there any issues that arose?
- Found that the minutes and seconds input fields were still accepting '.' so I added it to the invalid keys array.
### Is there anything that you need from your teammate?
- Please review my function and make sure it functions correctly!
### Refactor checklist
- [x] Check variable and parameter names: are they descriptive and unique?
- [x] Is the code DRY? SRP?
- [x] Is it easy for others to read your code? (explanatory comments needed?)
- [ ] Is the syntax consistent? (e.g. use of semicolons, blank lines)
- [ ] Are commit messages formatted correctly?

### CODE REVIEW

**Review by jasmyn2244 — APPROVED** [review]
What a sleek solution! I dig it!

File: index.html [hunk 1]
```diff
@@ -24,14 +24,17 @@ <h1 class="form-text-style">Select a category:</h1>
...
               <div class="minutes-input-label">
                 <label class="form-text-style" for="minutes">Minutes</label> <br>
-                <input type="text" class="minutes-input" name="minutes">
+                <input type="number" class="minutes-input" name="minutes">
+                <p class="error-message hidden" id="minutes-error"><img class="error-image" src="assets/warning.svg" alt="error icon"> Minutes are required.</p>
               </div>
               <div class="seconds-input-label">
                 <label class="form-text-style" for="seconds">Seconds</label> <br>
-                <input type="text" class="seconds-input" name="seconds">
+                <input type="number" class="seconds-input" name="seconds">
```
Comment (Jayquintana) [inline index.html:36]: Looks good!

File: main.js [hunk 2]
```diff
@@ -1,22 +1,40 @@
...
 var exerciseButton = document.querySelector('.exercise-button');
-var buttonImages = document.querySelectorAll('.category-button-icons')
+var buttonImages = document.querySelectorAll('.category-button-icons');
+var startActivityButton = document.querySelector('.start-activity-button');
+//inputs
+var accomplishInput = document.querySelector('.accomplish-input');
+var minutesInput = document.querySelector('.minutes-input');
+var secondsInput = document.querySelector('.seconds-input');
+var inputs = document.querySelectorAll('input');
+var errorMessages = document.querySelectorAll('.error-message');
```
Comment (Jayquintana) [inline main.js:16]: Good find here selecting them all !

File: main.js [hunk 3]
```diff
@@ -1,22 +1,40 @@
...
 });
 
 exerciseButton.addEventListener('click', function(event) {
-  hightlightButton(event)
+  hightlightButton(event);
 });
 
+startActivityButton.addEventListener('click', displayErrorMessage);
+
+minutesInput.addEventListener('keydown', preventEInput)
```
Comment (Jayquintana) [inline main.js:34]: small but needs semi colon

File: main.js [hunk 4]
```diff
@@ -29,3 +47,20 @@ function hightlightButton(event) {
...
   }
 }
+
+function preventEInput(event) {
+  if (invalidKeys.includes(event.key)) {
+    event.preventDefault();
+  }
+}
+
+function displayErrorMessage() {
```
Comment (Jayquintana) [inline main.js:57]: great work using the for loops!

File: styles.css [hunk 5]
```diff
@@ -22,6 +22,10 @@ body {
   background-color: #2B2733;
 }
 
+.hidden {
```
Comment (Jayquintana) [inline styles.css:25]: need to move this somewhere els

**Review by Jayquintana — COMMENTED** [review]
Good work team!

**Pull request merged**